nouveau/src/main/java/org/apache/couchdb/nouveau/api/DocumentDeleteRequest.java

```diff
@@ -24,24 +24,32 @@ public class DocumentDeleteRequest {
     @Positive
     private long seq;
 
+    private boolean purge;
+
     public DocumentDeleteRequest() {
         // Jackson deserialization
     }
 
-    public DocumentDeleteRequest(long seq) {
+    public DocumentDeleteRequest(long seq, final boolean purge) {
```

Comment:
Since we set the purge sequence once at the end after all the purges have been processed do we still need the `purge` field?